index.js

```diff
@@ -259,14 +268,16 @@ module.exports = function(THREE) {
                 // restrict phi to be betwee EPS and PI-EPS
                 phi = Math.max( EPS, Math.min( Math.PI - EPS, phi ) );
 
+                scale += scaleDelta * (( this.enableDamping === true ) ? this.dampingFactor : 1);
```

Comment:
We shouldn't rename scale to scaleDelta because it just isn't delta. 'scale' is the new zoomscale relative to the current zoomScale in the scope of the current 'update'. This is sort of the inverse of the current delta. scale + delta == 1

This works literally the same way rotation damping does. Therefore with the same side effects.
If a user starts using the mouse wheel while animation is in progress, we will have two independent zoom processes at the same time. So if we double-tap to zoom in and then immediately start zooming out using mouse wheel, zoom will go back and forth for a fraction of a second while the scaleDelta value is not completely damped out.